front/app/api/authentication/sign_in_out/signOut.ts

```diff
@@ -19,8 +22,9 @@ export default async function signOut() {
       const url = `${AUTH_PATH}/${provider}/logout?user_id=${sub}`;
       window.location.href = url;
     } else {
+      await resetMeQuery();
+      invalidateQueryCache();
```

Comment:
Just curious - do we need to await the reset of "me" separately first so that when all the cache is invalidated on the next line it doesn't make any requests using the old "me"?

Reply:
We are invalidating the rest of the cache but reseting `me` which means we don't keep the previous data in place during the refetch.